commands/deploy.go

```diff
@@ -70,15 +70,17 @@ func deploy(cmd *cobra.Command, args []string) {
 
 	if name != "" {
 		bravefile.PlatformService.Name = name
-	}
+	} else {
+    log.Fatal("missing unit name")
+  }
 
-	if unitCPU != "" {
-		bravefile.PlatformService.Resources.CPU = unitCPU
-	}
+	//if unitCPU != "" {
+	bravefile.PlatformService.Resources.CPU = unitCPU
```

Comment:
If I'm not mistaken, this will overwrite the limits if they are loaded from Bravefile.